modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy

```diff
@@ -746,4 +757,22 @@ def setBuildStatus(repoName, commit, message, state) {
   ]);
 }
 
+def runPublishingE2ETests() {
+  build(
+    job: "publishing-e2e-tests/${params.PUBLISHING_E2E_TESTS_BRANCH}",
+    parameters: [
+      [$class: "StringParameterValue",
+        name: "PUBLISHING_API_COMMITISH",
```

Comment:
I've updated this to take the app name, but would be good to chat about how best to implement this in the e2e tests Jenkinsfile.